### PR TITLE
please review

### PR DESCRIPTION
**Pull request by m-madelyn**

### CODE REVIEW

File: scoring.js [hunk 1]
```diff
@@ -35,9 +35,33 @@ function calculateReturnScore(player) {
...
+          return calculatePassingScore(player) + calculateRushingScore(player);
+          break;
+      case 'RB':
+          return calculateRushingScore(player) + calculateReceivingScore(player) + calculateReturnScore(player);
+          break;
+      case 'WR':
+          return calculateRushingScore(player) + calculateReceivingScore(player) + calculateReturnScore(player);
+          break;
+      case 'TE':
+          return calculateReceivingScore(player);
```
Comment (john-rice) [inline scoring.js]: Nicely done!

File: scoring.js [hunk 2]
```diff
@@ -35,9 +35,33 @@ function calculateReturnScore(player) {
   return parseFloat((kickYards + kickTouchdowns + kickFumbles + puntYards + puntTouchdowns + puntFumbles).toFixed(2))
 }
```
Comment (john-rice) [inline scoring.js]: No need to commit commented-out code.  Git provides us with the history of old code.  Please delete the commented out code.

**Comment by m-madelyn** [comment]
commented-out code deleted. I requested another review. Do I need another Pull Request?

**Comment by john-rice** [comment]
Well done.

**Pull request merged**